common/scripts/ci/run_boiler_app_integration_tests_with_live_backend.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash -i
+set -ex
+
+# temporary, wait for CI-infa
+export SDK_BACKEND=TIGER
+export FILTER=boilerapp.spec.ts
+export TIGER_DATASOURCES_NAME=vertica_staging-goodsales
+export TIGER_API_TOKEN=Z3JhcGhlbmUudGlnZXI6dGllbjowWVNaZVl1cEFXeWdzK3NTMVRMekJFTE1jV3hsbFZuSw==
+export TEST_BACKEND=https://staging-automation.anywhere.gooddata.com
+
+
+IMAGE_ID=gooddata-boiler-app
+APP_NAME=new-boiler-app
+APP_VERSION=app-toolkit@9.0.0-alpha.2
+APP_LANGUAGE=ts
+NODE_IMAGE='020413372491.dkr.ecr.us-east-1.amazonaws.com/tools/gdc-frontend-node-16:node-16.20.0-yarn-1.22.17'
+# CYPRESS_IMAGE='020413372491.dkr.ecr.us-east-1.amazonaws.com/tools/gdc-frontend-cypress-included:202212221405.4f8aee4'
+CYPRESS_IMAGE='cypress/included:12.10.0'
+
+
+DIR=$(echo $(cd $(dirname "${BASH_SOURCE[0]}") && pwd -P))
+ROOT_DIR=$(echo $(cd $(dirname "${BASH_SOURCE[0]}")/../../../ && pwd -P))
+E2E_TEST_DIR=$ROOT_DIR/libs/sdk-ui-tests-e2e
+_RUSH="${DIR}/docker_rush.sh"
+_RUSHX="${DIR}/docker_rushx.sh"
+sdk_backend=$(tr <<< "${SDK_BACKEND:?}" '[:upper:]' '[:lower:]')
+
+if [[ ! "${TEST_BACKEND:?}" =~ https?:// ]]; then
+    export TEST_BACKEND="https://${TEST_BACKEND}"
+fi
+
+pushd $E2E_TEST_DIR
+cat > .env <<-EOF
+SDK_BACKEND=${SDK_BACKEND:-TIGER}
+HOST=${TEST_BACKEND:-}
+CYPRESS_TEST_TAGS=checklist_integrated_${sdk_backend}
+FIXTURE_TYPE=goodsales
+FILTER=${FILTER:-}
+EOF
+
+if [[ "$SDK_BACKEND" == 'BEAR' ]]; then
+    cat >> .env <<-EOF
+USER_NAME=${USER_NAME:?}
+PASSWORD=${PASSWORD:?}
+AUTH_TOKEN=${AUTH_TOKEN:?}
+EOF
+
+elif [[ "$SDK_BACKEND" == 'TIGER' ]]; then
+    cat >> .env <<-EOF
+TIGER_API_TOKEN=${TIGER_API_TOKEN:?}
+TIGER_DATASOURCES_NAME=${TIGER_DATASOURCES_NAME:?}
+EOF
+else
+    echo "Unknown SDK_BACKEND=${SDK_BACKEND}. Exiting..."
+    exit 1
+fi
+
+# $_RUSH install
+# $_RUSH build -t sdk-ui-tests-e2e
+$_RUSHX libs/sdk-ui-tests-e2e create-ref-workspace
+
+export $(cat .env)
+export HOST=https://127.0.0.1:8080
+
+pushd "${ROOT_DIR}" || exit 1
+
+docker run --rm --entrypoint '' \
+-e APP_NAME=$APP_NAME \
```

Comment:
if the variable is environment variable, there is no need to remap like this, using this declaration instead
`-e APP_NAME \ `